.github/workflows/unit-tests.yml

```diff
@@ -9,8 +9,27 @@ on:
       - main
 
 jobs:
-  unit-test:
-    if: github.repository_owner == 'rancher'
+  detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      kubewarden: ${{ steps.filter.outputs.kubewarden }}
+      sbomscanner: ${{ steps.filter.outputs.sbomscanner }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Detect changed directories
+        uses: dorny/paths-filter@v3
```

Comment:
I think this won't work here, see
https://github.com/rancher/security-team/blob/c6b7143384a3c22b9950deb34913a213d5019f49/docs/standards/rancher-gha-standards.md#previously-rejected

You can use only actions from allow list here, could you double check it will work?
https://github.com/rancher/security-team/issues/1371

Reply:
Also you can just use `path` for this.
https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#example-including-paths

Reply:
Yes, which is pretty much the same of your action if I got it right. In this way it's set ahead and will not be triggered and avoid to pollute the CI.